modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -1952,6 +1952,32 @@ m|+++false+++
 |===
 
 
+[[config_dbms.cypher.infer_schema_parts]]
+=== `dbms.cypher.infer_schema_parts`
+
+.dbms.cypher.infer_schema_parts
```

Comment:
The same should be added to the code base.

Reply:
This is what I try to get in:
https://github.com/neo-technology/neo4j/pull/25555/files#diff-22e5a6968aee5ed485e16da8650e784a16dbd787005216bb719d3dc44aa2b43aR249

Reply:
That PR has been merged but the check still fails.